modules/sql/src/main/scala/SqlMapping.scala

```diff
@@ -160,6 +160,7 @@ trait SqlMapping[F[_]] extends CirceMapping[F] with SqlModule[F] { self =>
           case u: UntypedNarrow => u.copy(child = loop(u.child, context))
           case n@Narrow(subtpe, _) => n.copy(child = loop(n.child, context.asType(subtpe)))
           case s: Skip => s.copy(child = loop(s.child, context))
+          case o: Offset => loop(o.child, context)
```

Comment:
I think we can eliminate some other constructors here (like `Limit` and `OrderBy`) since they're handled by the database. We _must_ do it for `Offset` otherwise we will `drop` results we actually want to see!

Reply:
I think we may need to retain `OrderBy` ... I think if we have multiple nested orderings there's no guarantee that the eventual order of rows internally is going to be preserved in the right way. But agreed that we _ought_ to be able to eliminate `Limit`.